docs/content/topics/scripting.md

```diff
@@ -63,7 +63,7 @@ message to the log:
 ```javascript
 console.log("hello world")
 ```
-[brigade-01.js](examples/brigade-01.js)
+[brigade-01.js](../../examples/brigade-01.js)
```

Comment:
It seems like this should be `../examples/brigade-01.js`? (And others as well

Reply:
yeah, that's the way the new Netlify platform works, if you hit them in the .md file they will not work, they do work though once they are rendered, check out both previews

- https://deploy-preview-885--brigade-docs.netlify.com/topics/scripting/
- https://deploy-preview-885--brigade-docs.netlify.com/topics/scripting_advanced/

Reply:
Gotcha! Thanks ☺️